### PR TITLE
bruno: fix darwin builds

### PR DESCRIPTION
**Pull request by mattpolzin**
## Description of changes

Fix Darwin builds of the `bruno` package.

I'm a bit out of my depth here so not confident these changes are ideal, I just know that they do indeed work. Prior to these changes, builds on Darwin failed with very unhelpful errors from the `electron-builder` script, after these changes you get a working application bundle.

One thing I am very unclear on is the right way to track this added support for Darwin against the package. The package inherits `platforms` from `electron.meta` which feels very appropriate, but the `electron` derivation will be one of two different packages depending on the platform you evaluate it on; for Darwin, you get the binary electron package and for everything else you get the source code electron package. This means that the GitHub labeling will indicate this derivation does not cause any Darwin rebuilds but at the same time if I build `bruno` on my Darwin machine I don't need to do anything special (no overrides or overlays needed) for Nix to understand that my platform is supported. Pretty awkward.

Unrelatedly, it would not build for arm64 linux because the output path was `linux-arm64-unpacked` instead of `linux-unpacked` so I fixed that in a second commit.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by lucasew** [comment]
Result of `nixpkgs-review pr 280825` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>bruno</li>
  </ul>
</details>

**Comment by lucasew** [comment]
Diff looks good to me.

BTW could you add yourself as maintainer too? That way when someone else sends a PR changing this derivation you also get pinged and can test if it's still working on Darwin. I have only two x86_64-linux machines myself.

**Review by lucasew — APPROVED** [review]

**Review by kashw2 — APPROVED** [review]

**Review by water-sucks — APPROVED** [review]

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -11603,6 +11603,12 @@
     githubId = 279868;
     name = "Matti Kariluoma";
   };
+  mattpolzin = {
+    email = "matt.polzin@gmail.com";
+    github = "mattpolzin";
+    githubId = 2075353;
```
Comment (fabianhjr) [inline maintainers/maintainer-list.nix:11609]: Checked that https://api.github.com/users/mattpolzin matches

**Comment by fabianhjr** [comment]
For future reference, there is another Discourse thread something along the lines of "PRs already reviewied" where nixpkgs committers help with merging if something has already been reviewed.

Thanks for contributing! UwU

**Pull request merged**